modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/routes/AssetsRoutes.scala

```diff
@@ -0,0 +1,30 @@
+package org.ergoplatform.explorer.http.api.v0.routes
+
+import cats.effect.{ContextShift, Sync}
+import org.ergoplatform.explorer.http.api.syntax.applicativeThrow._
+import org.ergoplatform.explorer.http.api.v0.services.AssetsService
+import org.http4s.HttpRoutes
+import sttp.tapir.server.http4s._
+
+final class AssetsRoutes[F[_]: Sync: ContextShift](
+  service: AssetsService[F, fs2.Stream]
+) {
+
+  import org.ergoplatform.explorer.http.api.v0.defs.AssetsEndpointDefs._
+
+  val routes: HttpRoutes[F] =
+    getAllIssuingBoxesR
+
+  private def getAllIssuingBoxesR: HttpRoutes[F] =
+    getAllIssuingBoxesDef.toRoutes { _ =>
+      service.getAllIssuingBoxes.compile.toList.either
```

Comment:
Why compiling stream here, we are loosing all gained earlier benefits doing this. Http4s supports streaming.

Reply:
I saw [`// API v0 format does not allow to use streaming`](http://github.com/ergoplatform/explorer-backend/blob/bb967e951790aa44da188b962953da034980f8df/modules/explorer-api/src/main/scala/org/ergoplatform/explorer/http/api/v0/routes/BlocksRoutes.scala#L30-L30) comment. There are no endpoints with streaming in the code and I could not make one.

Reply:
Right, many of existing methods from APIv0 doesn't allow to use streaming due to response structure, but you're adding new method to APIv0 and streaming can easily be used here.
You can find answers here https://tapir-scala.readthedocs.io/en/latest/server/http4s.html